samples/cellular/modem_shell/src/fota/fota_shell.c

```diff
@@ -56,7 +64,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 	sub_fota,
 	SHELL_CMD_ARG(
 		download, NULL,
-		"<server> <filename>\nDownload and install a FOTA update. Available servers are \"eu\", \"us\", \"jpn\" and \"au\".",
+		"[<server> <filename>]/[<uri>]\nDownload and install a FOTA update. Available "
+		"servers are \"eu\", \"us\", \"jpn\" and \"au\"",
```

Comment:
I think `|` means OR rather than `/` (and I'd use spaces for clarity):
```suggestion
		"[<server> <filename>] | [<uri>]\nDownload and install a FOTA update. "
		"Available servers are \"eu\", \"us\", \"jpn\" and \"au\"",
```
Moving "Available" also to next line making it a little nicer to read from the code.

Reply:
Moved.